### PR TITLE
Fix Functional Tests on Travis

### PR DESCRIPTION
**Pull request by helhum**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
   ],
   "license": "GPL-2.0+",
   "require": {
-    "typo3/cms": ">=6.2.0,<9.0.0"
+    "typo3/cms-core": ">=6.2.0,<9.0.0"
```
Comment (IchHabRecht) [inline composer.json:21]: requiring typo/cms-core errors with message:

D:\Version Control\Git\TYPO3\Extensions\integrity>C:\Unix\PHP-5.5.28\php.exe "D:\Version Control\Composer\composer.phar" install
Loading composer repositories with package information
Installing dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - The requested package typo3/cms-core could not be found in any version, there may be a typo in the package name.

Potential causes:
- A typo in the package name
- The package is not available in a stable-enough version according to your minimum-stability setting
  see https://getcomposer.org/doc/04-schema.md#minimum-stability for more details.

Read https://getcomposer.org/doc/articles/troubleshooting.md for further common problems.

Comment (helhum) [inline composer.json:21]: The point is to have a requirement to `typo3/cms-core` in this package that the `composer require typo3/cms="$TYPO3_VERSION"` actually checks if the package is compatible with this requested TYPO3 version.

This means: you can never do a `composer install` with this composer.json, which otoh is also not what you want for testing (against multiple TYPO3 versions)

**Comment by helhum** [comment]
And as you can see in the travis build for this pull request, it works nicely.
But this is just a matter of style. I can change that back if you like me to.

**Comment by IchHabRecht** [comment]
Thank you for your help. Finally Travis CI is able to run the functional tests as well. Committed the changes independently.

**Pull request closed without merging**